remote-api/src/main/java/org/alfresco/rest/api/impl/NodesImpl.java

```diff
@@ -1848,6 +1848,19 @@ public Node createNode(String parentFolderNodeId, Node nodeInfo, Parameters para
         {
             versionMajor = Boolean.valueOf(str);
         }
+        String versioningEnabledStringValue = parameters.getParameter("versioningEnabled");
+        if(null != versioningEnabledStringValue)
```

Comment:
We normally have a space after the "if"

Reply:
Thank you for spotting this, fixed it in the last commit.